torch_xla/csrc/helpers.cpp

```diff
@@ -368,10 +368,9 @@ xla::XlaOp XlaHelpers::FlattenDimRange(xla::XlaOp input, int64_t start,
 xla::XlaOp XlaHelpers::LinearInterpolation(xla::XlaOp value0, xla::XlaOp value1,
                                            double alpha) {
   const xla::Shape& shape = XlaHelpers::ShapeOfXlaOp(value0);
-  xla::XlaOp one = xla::One(value0.builder(), shape.element_type());
   xla::XlaOp alpha_value =
       ScalarValue(alpha, shape.element_type(), value0.builder());
-  return value0 * alpha_value + value1 * (one - alpha_value);
+  return value0 + alpha_value * (value1 - value0);
```

Comment:
Thanks for the fix! Can you update the header as well?

https://github.com/pytorch/xla/blob/cc43bf117fd795443116a0b13a467f7f2fb35a91/torch_xla/csrc/helpers.h#L110